meson.build

```diff
@@ -0,0 +1,67 @@
+project(
+  'flatpak-builder',
+  'c',
+  license: 'LGPL-2.1-or-later',
+  meson_version: '>= 0.56.2',
+  version: '1.3.1',
+  default_options: 'c_std=gnu99',
+)
+
+cc = meson.get_compiler('c')
+
+add_project_arguments(
+  '-Werror=empty-body',
+  '-Werror=strict-prototypes',
+  '-Werror=missing-prototypes',
+  '-Werror=implicit-function-declaration',
+  '-Werror=format=2',
+  '-Werror=format-security',
+  '-Werror=format-nonliteral',
+  '-Werror=pointer-arith',
+  '-Werror=init-self',
+  '-Werror=missing-declarations',
+  '-Werror=return-type',
+  '-Werror=overflow',
+  '-Werror=int-conversion',
+  '-Werror=parentheses',
+  '-Werror=incompatible-pointer-types',
+  '-Werror=misleading-indentation',
+  '-Werror=missing-include-dirs',
+  # Necessary for non POSIX stuff in glibc like fallocate
+  '-D_GNU_SOURCE',
+  language: 'c',
+)
+
+dwarf_header_dir = get_option('dwarf_header_dir')
+
+inc_dirs = include_directories(
+  '.',
+# FIXME: handle $prefixes/libdwarf/ too or let it to the option? The later would be easier
+  # Dwarf header
+  dwarf_header_dir,
+)
+
+if not cc.has_header('dwarf.h', include_directories: inc_dirs)
+  error('dwarf.h header could not be found in either the include directories or @0@'.format(dwarf_header_dir))
+endif
+
+debugedit = find_program('debugedit')
```

Comment:
What is the function of this? You don't seem to do anything with it.

Reply:
Just forcing the build system to fail if debugedit is not present. Unlike the Autotools build, system-installed debugedit is a hard dependency for Meson.

Reply:
Right, but just because something exists on the build system doesn't mean it is available on the target system. I would remove this check and let flatpak-builder fail gracefully if the tool isn't found. This is the discussion we previously had in #mesonbuild